composer.json

```diff
@@ -2,6 +2,12 @@
   "name": "elabftw/elabftw",
   "description": "Laboratory notebook for research labs",
   "homepage": "https://www.elabftw.net",
+  "repositories": [
```

Comment:
can you explain the need for this? And why is eluceo/ical present below then?

Reply:
This is the [documented](https://getcomposer.org/doc/05-repositories.md#loading-a-package-from-a-vcs-repository) way to include an own fork of a third party library.

Ideally, the contend from my fork would go into eluceo iCal 2, but for the time being, I have little hope it will happen soon as [I did not get any answer from the maintainer](https://github.com/markuspoerschke/iCal/discussions/640#discussion-7274038).

Reply:
Yeah, I really have a problem with adding dependencies. It seems nearly all projects get abandoned at some point. And this one is a good example. No reviews or comments on PRs. Vulns are not handled (https://github.com/markuspoerschke/iCal/issues/517). So basically we would need to maintain a fork right from the start. Not a good look but I'm guessing this lib is pretty essential to the feature, right? All the commits since July are dependabot. There are also issues like https://github.com/markuspoerschke/iCal/issues/541 which seems pretty important to fix, but they seem ignored... Overall, really not a big fan of having to depend on yet another undermaintained php lib with no security policy or quality process...